src/content/ytui-music.mdx

```diff
@@ -0,0 +1,7 @@
+---
+title: YouTube UI Music
+excerpt: Listen to youtube from terminal ui
+tags: rust, tui, terminal, youtube, music
+featuredImg: '../images/sudipghimire533/image.png'
```

Comment:
The image name & extension doesn't matches with the one on your another PR for image. Please try to update the image on this same PR rather than creating a new one.